### ISSUE TITLE
AccessibilityInfo isScreenReaderEnabled

### ISSUE THREAD

**Issue by SavoySchuler** [microsoft/react-native-windows#3115]
https://facebook.github.io/react-native/docs/accessibilityinfo.html#isscreenreaderenabled
Not supported in current

**Comment by chrisglein** [comment]
Keeping current bar for M4, so this can move to M5.

**Comment by NickGerleman** [comment]
Should be implemented by checking is UIA clients are listening in https://github.com/microsoft/react-native-windows/blob/master/vnext/Microsoft.ReactNative/Modules/AccessibilityInfoModule.cpp
